datastore/transaction.go

```diff
@@ -307,7 +309,13 @@ func (t *Transaction) GetMulti(keys []*Key, dst interface{}) (err error) {
 	opts := &pb.ReadOptions{
 		ConsistencyType: &pb.ReadOptions_Transaction{Transaction: t.id},
 	}
-	return t.client.get(t.ctx, keys, dst, opts)
+
+	// TODO: Use transaction ID returned by get
+	_, err = t.client.get(t.ctx, keys, dst, opts)
+	if err != nil {
+		return err
+	}
+	return nil
```

Comment:
This is just a general Golang question. Would return err instead of the if and then return nil do the same thing?

Reply:
Both are fine in my opinion, but you'd probably see `return err` without the `if` more often in cases like this.

Reply:
Fixed!